copyDynamoTable.js

```diff
@@ -0,0 +1,44 @@
+const { copy } = require('copy-dynamodb-table');
+
+const globalAWSConfig = {
+  // AWS Configuration object http://docs.aws.amazon.com/AWSJavaScriptSDK/latest/AWS/Config.html#constructor-property
+  accessKeyId: 'your_access_key',
```

Comment:
same here with these variables.

Also, I see this `gobalAWSConfig` used in a couple of places. Would it be worth moving that to its own file?

Reply:
Moved to another file